### PR TITLE
Add persona plugins

### PR DESCRIPTION
**Pull request by NeonDaniel**

### CODE REVIEW

File: ovos_plugin_manager/persona.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    Get relevant configuration for factory methods
+    @param config: global Configuration OR plugin class-specific configuration
+    @return: plugin class-specific configuration
+    """
+    from ovos_plugin_manager.utils.config import get_plugin_config
+    config = config or Configuration()
+    return get_plugin_config(config, "persona")
+
+
+class OVOSPersonaFactory:
```
Comment (JarbasAl) [inline ovos_plugin_manager/persona.py:81]: this should not be here, there is a persona service that manages personas, like there is a skills service for skills and no factory

File: ovos_plugin_manager/templates/persona.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from ovos_bus_client.message import dig_for_message
+from ovos_utils import classproperty
+from ovos_utils.process_utils import RuntimeRequirements
+from quebra_frases import span_indexed_word_tokenize, word_tokenize
+
+
+class Persona:
```
Comment (JarbasAl) [inline ovos_plugin_manager/templates/persona.py:7]: no class needed, personas are .json files

https://github.com/OpenVoiceOS/ovos-persona/tree/dev/personas

Comment (NeonDaniel) [reply ovos_plugin_manager/templates/persona.py:7]: I'm not sure I understand the entrypoints here then; if its just configuration, what would a persona package be providing? Feel free to push to this branch directly

Comment (JarbasAl) [reply ovos_plugin_manager/templates/persona.py:7]: user provided personas are .json files

package provided personas are a dict, which OPM can then scan, literally just need to delete this class

**Review by JarbasAl — APPROVED** [review]

**Pull request merged**